src/BuildScriptGenerator/SourceRepo/SourceRepoFileHelpers.cs

```diff
@@ -0,0 +1,18 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Oryx.BuildScriptGenerator.SourceRepo
+{
+    internal static class SourceRepoFileHelpers
+    {
+        internal static dynamic ReadJsonObjectFromFile(ISourceRepo sourceRepo, string fileName)
```

Comment:
sugg: we could make this an extension method, to make it easier to find?

Reply:
good idea